packages/components/src/Enumeration/Enumeration.scss

```diff
@@ -7,18 +7,25 @@ $tc-enumeration-font-size: 1.4rem !default;
 	display: flex;
 	flex-direction: column;
 
-	:global(input[type=text]) {
-		font-size: $tc-enumeration-font-size;
-		line-height: $tc-enumeration-font-size;
-	}
+	:global {
+		input[type='text'] {
+			font-size: $tc-enumeration-font-size;
+			line-height: $tc-enumeration-font-size;
+		}
+
+		.btn {
+			justify-content: flex-start;
+			min-height: auto;
 
-	:global(.btn.btn-link) {
-		padding: 0 $padding-smaller;
+			.btn-link {
```

Comment:
.btn-link is not supposed to be a sub item class, I guess you want &.btn-link here